### PR TITLE
gxml: fix pc config and declare indirect deps with linkage

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----
https://github.com/Homebrew/homebrew-core/actions/runs/10259102515/job/28405420278?pr=180172#step:3:9678
https://gitlab.gnome.org/GNOME/gxml/-/merge_requests/28

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  ==> pkg-config --cflags --libs gxml-0.20
  Package 'gxml-0.20' requires 'libxml-2.0 >= 2.9.13' but version of libxml-2.0 is 2.9.4
```

File: Formula/g/gxml.rb [hunk 1]
```diff
@@ -39,35 +51,23 @@ def install
...
+--- a/gxml/gxml.pc.in
++++ b/gxml/gxml.pc.in
+@@ -8,6 +8,6 @@ Name: libgxml
+ Description: GObject API for XML manipulation and serialization library
+ URL: http://live.gnome.org/GXml
+ Version: @PROJECT_VERSION@
+-Requires: glib-2.0 >= 2.72 gio-2.0 >= 2.72 gobject-2.0 >= 2.72 libxml-2.0 >=2.9.13 gee-0.8 >=0.20.6
++Requires: glib-2.0 >= 2.72 gio-2.0 >= 2.72 gobject-2.0 >= 2.72 libxml-2.0 >= 2.9.13 gee-0.8 >= 0.20.6
+ Libs: -L${libdir} -lgxml-@API_VERSION@
+ Cflags: -I${includedir}/gxml-@API_VERSION@
```
Comment (carlocab) [inline Formula/g/gxml.rb:73]: Looks unused?

Comment (chenrui333) [reply Formula/g/gxml.rb:73]: yeah, that is right :)

**Review by p-linnane — APPROVED** [review]

File: Formula/g/gxml.rb [hunk 2]
```diff
@@ -4,6 +4,7 @@ class Gxml < Formula
   url "https://gitlab.gnome.org/GNOME/gxml/-/archive/0.20.4/gxml-0.20.4.tar.bz2"
   sha256 "d8d8b16ff701d0c5ff04b337b246880ec4523abfe897a1f77acdf7d73fb14b84"
   license "LGPL-2.1-or-later"
+  revision 1
```
Comment (carlocab) [inline Formula/g/gxml.rb:7]: Is this needed?

**Pull request merged**